nova_vm/src/ecmascript/builtins/text_processing/regexp_objects/regexp_prototype.rs

```diff
@@ -179,13 +181,150 @@ impl RegExpPrototype {
         todo!()
     }
 
+    /// ### [22.2.6.4 get RegExp.prototype.flags](https://tc39.es/ecma262/#sec-get-regexp.prototype.flags)
     fn get_flags(
-        _agent: &mut Agent,
-        _this_value: Value,
+        agent: &mut Agent,
+        this_value: Value,
         _: ArgumentsList,
-        _gc: GcScope<'_, '_>,
+        mut gc: GcScope<'_, '_>,
     ) -> JsResult<Value> {
-        todo!()
+        // 1. Let R be the this value.
+        let r = this_value;
+
+        // 2. If R is not an Object, throw a TypeError exception.
+        if !r.is_object() {
```

Comment:
```suggestion
        let Ok(r) = Object::try_from(t) else {
```

This lets you avoid the unwraps and such later.

Reply:
Thank you for the review! I fixed it in https://github.com/trynova/nova/pull/496/commits/fe908ba8060f7863f3a6c4ac24bb86e6974f5660!